assembly/src/assembler/mod.rs

```diff
@@ -313,37 +312,26 @@ impl Assembler {
 
 /// Compilation/Assembly
 impl Assembler {
-    /// Compiles the provided module into a [`MastForest`].
-    ///
-    /// # Errors
-    ///
-    /// Returns an error if parsing or compilation of the specified program fails.
-    pub fn assemble(self, source: impl Compile) -> Result<MastForest, Report> {
```

Comment:
It seems the Assembler was changed so that it is no longer possible to assemble a library, only programs. The compiler, for example, is going to primarily be assembling libraries, _not_ programs - particularly for the rollup, where accounts are libraries, and note scripts are not executables but libraries dynamically invoked from the "real" executable, the transaction kernel.

How does one obtain a `MastForest` for a library from the `Assembler`? I took a look around, but it looks to me like it isn't possible.